errbot-reminder.py

```diff
@@ -0,0 +1,82 @@
+from datetime import date, datetime, time, timedelta
+
+from errbot import BotPlugin, botcmd
+
+# Daily / Retrospective --> If it's an ordinary day or a retrospective, meeting at 9:30
+# Sprint Planning --> If it's a Sprint planning, meeting at 15:30
+# Sprint review --> If it's a Sprint Review, meeting at 14:45
+#  -----------------
+
+
+class Reminder(BotPlugin):
+    @botcmd
+    def whereami(self, msg, args):
+        stream = msg._from._room._id
+        topic = msg._from._room._subject
+
+        client = self._bot.client
+
+        first_iteration_startdate = datetime(2022, 2, 28)
+
+        def get_monday():
+            today_weekday = date.today().weekday()
+            monday = date.today() - timedelta(days=today_weekday)
+            monday = datetime.combine(monday, datetime.min.time())
+            return monday
+
+        def is_sprint_planning():
+            nb_days = first_iteration_startdate - get_monday()
+            nb_weeks = nb_days / 7
+            nb_weeks = int(nb_weeks.total_seconds())
+
+            if nb_weeks % 2 == 0:
+                return True
+
+        day_of_the_week = datetime.now().weekday()
+        current_time = datetime.now().replace(second=0, microsecond=0)
+        stream = "tools & services"
+
+        if day_of_the_week != 5 and day_of_the_week != 6:
+            if date.today().weekday() == 0 and is_sprint_planning:
+                meeting_time = time(15, 30)
+                topic = "sprint planning"
+
+            elif date.today().weekday() == 3 and not is_sprint_planning:
```

Comment:
the `is_sprint_planning` is never called

Reply:
This applies to all the code

Reply:
I call this function at the lines 40, 44 and 48. But I forgot to put the brackets, sorry